src/components/editor/PropertyTemplate.jsx

```diff
@@ -7,23 +7,29 @@ class PropertyTemplate extends Component {
       border: '1px dotted',
       padding: '10px'
     }
-    return (
-      <div className='PropertyTemplate' style={dottedBorder}>
-        <h5>BEGIN PropertyTemplate elements</h5>
-        <ul>
-          <li>propertyLabel: <strong>{this.props.propertyTemplates[0].propertyLabel}</strong></li>
-          <li>propertyURI: <strong>{this.props.propertyTemplates[0].propertyURI}</strong></li>
-          <li>type: <strong>{this.props.propertyTemplates[0].type}</strong></li>
-          <li>mandatory: <strong>{this.props.propertyTemplates[0].mandatory}</strong></li>
-          <li>repeatable: <strong>{this.props.propertyTemplates[0].repeatable}</strong></li>
-          <li>contained resourceTemplates[] length: <strong>{this.props.propertyTemplates[0].resourceTemplates.length}</strong></li>
-          <li>
-            <ValueConstraints {...this.props.propertyTemplates[0].valueConstraint} />
-          </li>
-        </ul>
-        <h5>END of PropertyTemplate elements</h5>
-      </div>
-    )
+    if (this.props.propertyTemplates.length == 0) {
+      return <h1>We should have propertyTemplates but we don't.</h1>
+    } else if (this.props.propertyTemplates[0].resourceTemplates == undefined ) {
```

Comment:
Can we just use `if(!this.props.propertyTemplates)` here for both `length == 0` and `'undefined'` ?

Reply:
didn't think of it - good idea!

Reply:
@jgreben and I discussed F2F;  plan is to use #158 ultimately

Reply:
👌